modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/SearchBar.js

```diff
@@ -61,38 +54,48 @@ export default function SearchBar({
 	const alignElementRef = useRef();
 	const dropdownRef = useRef();
 
-	const {resource} = useResource({
-		fetchOptions: {
-			body: suggestionsContributorConfiguration,
-			credentials: 'include',
-			headers: new Headers({
-				'Accept': 'application/json',
-				'Accept-Language': Liferay.ThemeDisplay.getBCP47LanguageId(),
-				'Content-Type': 'application/json',
-				'x-csrf-token': Liferay.authToken,
-			}),
-			method: 'POST',
-		},
-		fetchPolicy: 'cache-first',
-		link: fetchURL.href,
-		onNetworkStatusChange: (status) => {
-			setNetworkState({
-				error: status === 5,
-				loading: status > 1 && status < 4,
-				networkStatus: status,
+	const _handleFetchSuggestions = (value) => {
+		fetch(
+			addParams(
+				{
+					currentURL: window.location.href,
+					destinationFriendlyURL: destinationFriendlyURL.trim().length
+						? destinationFriendlyURL
+						: '/search',
+					groupId: Liferay.ThemeDisplay.getScopeGroupId(),
+					plid: Liferay.ThemeDisplay.getPlid(),
+					scope,
+					search: value,
+				},
+				suggestionsURL
+			),
+			{
+				body: suggestionsContributorConfiguration,
+				headers: new Headers({
+					'Accept': 'application/json',
+					'Accept-Language': Liferay.ThemeDisplay.getBCP47LanguageId(),
+					'Content-Type': 'application/json',
+				}),
+				method: 'POST',
+			}
+		)
+			.then((response) => response.json())
+			.then((data) => {
+				setLoading(false);
+
+				setResourceItems(data?.items || []);
+			})
+			.catch(() => {
+				setLoading(false);
+
+				setResourceItems([]);
 			});
-		},
-		variables: {
-			currentURL: window.location.href,
-			destinationFriendlyURL: destinationFriendlyURL.trim().length
-				? destinationFriendlyURL
-				: '/search',
-			groupId: Liferay.ThemeDisplay.getScopeGroupId(),
-			plid: Liferay.ThemeDisplay.getPlid(),
-			scope,
-			search: autocompleteSearchValue,
-		},
-	});
+	};
+
+	const [handleFetchSuggestionsDebounced] = useDebounceCallback(
```

Comment:
Rename this as well `handleFetchSuggestionsDebounced` to `fetchSuggestionsDebounced`